.changeset/afraid-gifts-protect.md

```diff
@@ -0,0 +1,20 @@
+---
+'@hashicorp/react-accordion': patch
+'@hashicorp/react-callouts': patch
+'@hashicorp/react-case-study-slider': patch
+'@hashicorp/react-checkbox-input': patch
+'@hashicorp/react-consent-manager': patch
+'@hashicorp/react-enterprise-alert': patch
+'@hashicorp/react-featured-slider': patch
+'@hashicorp/react-learn-callout': patch
+'@hashicorp/react-logo-grid': patch
+'@hashicorp/react-min-100-layout': patch
+'@hashicorp/react-search': patch
+'@hashicorp/react-subnav': patch
+'@hashicorp/react-tabbed-accordion': patch
+'@hashicorp/react-tabs': patch
+'@hashicorp/react-text-input': patch
+'@hashicorp/react-toggle': patch
+---
+
+Removes package-lock.json
```

Comment:
I'm not sure these releases are really necessary, but `changesets` thinks they are, and I trust that 👍